src/Node/Api.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the KongAdminApi package.
+ *
+ * (c) Unikorp <https://github.com/unikorp>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Unikorp\KongAdminApi\Node;
+
+use Unikorp\KongAdminApi\AbstractNode;
+use Unikorp\KongAdminApi\Document\Api as Document;
+
+/**
+ * api
+ *
+ * @author VEBER Arnaud <https://github.com/VEBERArnaud>
+ */
+class Api extends AbstractNode
+{
+    /**
+     * add api
+     *
+     * @param \Unikorp\KongAdminApi\Document\Api $document
```

Comment:
I tried running [sami](https://github.com/FriendsOfPHP/Sami) on the project, and it works like a charm

<img width="1233" alt="screen shot 2017-01-29 at 9 08 19 pm" src="https://cloud.githubusercontent.com/assets/5168814/22407478/556f64c8-e667-11e6-8893-9b55dc1050e4.png">